main.cpp

```diff
@@ -183,105 +302,264 @@ void Vinyl::moveToneArm(bool is12Inch)
 {
     float distanceToMoveArm;
     distanceToMoveArm = is12Inch ? 6.0f : 3.5f; 
+    std::cout << "Move tone arm " << distanceToMoveArm << "inches" << std::endl;
+}
+
+double Vinyl::bringJoy(int setNumberOfGoodTracks, double setListenerMood)
+{
+    double quality = double(setNumberOfGoodTracks) * setListenerMood;
+    return quality;
+}
+
+double Vinyl::accrueValue(int setnumberOfGoodTracks, int playsOnYoutube)
+{
+    double valueScale = 0.00005;
+    double value = double(setnumberOfGoodTracks) * double(playsOnYoutube);
+    return value * valueScale;
 }
 
 struct Oscillator
 {
-    double sawLevel = 0.0;
-    double squLevel = 1.0;
-    double tuning = 0.0;
-    bool subOscOn = false;
-    int octave = 0; 
+    Oscillator();
+    double sawLevel;
+    double squLevel;
+    double tuning;
+    bool subOscOn;
+    int octave;
 
     double osc1FM(double osc1SawSignal, double osc1SquSignal);
-    double subOscSignal(bool subOscOn);
+    double subOscSignal(bool setDubOscOn);
```

Comment:
change this variable name to not contain a verb
Name things what they are.
Name functions based on the action they perform
Variables represent things. 
functions represent actions.
- [ ] revise